soc/README.md

```diff
@@ -0,0 +1,53 @@
+Summer of Code
+=====================
+
+Organization Admins
+-------------------
+
+If you need help with anything Summer of code related, you can file an issue or
+contact one of the admins below:
+
+- Michelle Noorali ([@michellenoorali](https://github.com/michelleN)\):
+  [https://twitter.com/michellenoorali](https://twitter.com/michellenoorali)
+- Thomas Rampelberg ([@grampelberg](https://github.com/grampelberg)\):
+  [https://twitter.com/grampelberg](https://twitter.com/grampelberg)
+
+Communication
+-------------
+
+Please reach out to us on the [SMI slack](https://aka.ms/smi/slack) in the
+#general channel.
```

Comment:
lint is complaining about this -__-
maybe put #general in back tics `#general` ?